configs/disco/README.md

```diff
@@ -0,0 +1,149 @@
+# Disco Diffusion(Google Colab)
+
```

Comment:
this folder should be named as `disco_diffusion` rather than `disco` to align with the implementation of models.